docs/howto/openstack/nova/resize-server.md

```diff
@@ -0,0 +1,131 @@
+
+    ![A panel displaying the servers current flavor, 8 Cores and 24 GB RAM with the name of the flavor "b.8c24gb" within parenthesis.](assets/resize-server/04-modify-server.png)
+=== "OpenStack CLI"
+    To list all available flavors you can simply run `openstack flavor list`, but that will return a very long unsorted list, instead we recommend the following command:
+
+    ```bash
+    openstack flavor list -c Name -f value | grep '[0-9]c[0-9]' | sort -V
+    ```
+
+    The printout is a simple and clean list, sorted by the compute type, the number of cores and then by the amount of memory.
+
+    ```plain
+    b.1c1gb
+    b.1c2gb
+    b.1c4gb
+    b.2c2gb
+    b.2c4gb
+    b.2c8gb
+    b.2c16gb
+    b.4c4gb
+    ...
+    ```
+
+## Initiating the resize
+
+Choose a new flavor that you want your server to use instead.
+
+=== "{{gui}}"
+    Once selected, the _Resize_ button will appear.
+
+    ![A panel displaying the servers new flavor, 4 Cores and 24 GB RAM with the name of the flavor "b.4c24gb" within parenthesis, also a green button below it with the text "Resize".](assets/resize-server/05-resize-button.png)
+
+    Click on it to start the resize.
+
+    While the resize is ongoing you will see a spinning circle saying _Resize is in progress_.
+
+    ![A spinning circle with the text "Resize in progress...".](assets/resize-server/06-resize-in-progress.png)
+=== "OpenStack CLI"
+    A resize is only possible with flavors using the same prefix letter, most commonly you will have a `b.` flavor and therefore you must select another `b.` flavor.
+
+    To start the resize use the following command:
+
+    ```bash
+    openstack server resize --flavor <new_flavor> <server_id>
+    ```
+
+    While the resize is ongoing the server should have the OS-EXT-STS:task_state of `resize_migrating` and the status of `RESIZE`.
+
+    ```bash
+    openstack server show -c OS-EXT-STS:task_state -c OS-EXT-STS:vm_state -c status <server_id>
+    ```
+
+    ```plain
+    +-----------------------+------------------+
+    | Field                 | Value            |
+    +-----------------------+------------------+
+    | OS-EXT-STS:task_state | resize_migrating |
+    | OS-EXT-STS:vm_state   | active           |
+    | status                | RESIZE           |
+    +-----------------------+------------------+
+    ```
+
+    You may proceed with the next step once your server status is `VERIFY_RESIZE`.
+
+    ```plain
+    +-----------------------+---------------+
+    | Field                 | Value         |
+    +-----------------------+---------------+
+    | OS-EXT-STS:task_state | None          |
+    | OS-EXT-STS:vm_state   | resized       |
+    | status                | VERIFY_RESIZE |
+    +-----------------------+---------------+
+    ```
+
+The resize process might take a minute or more. {{brand}} Cloud will now make a restore point incase the resize process fails, it would then restore your server to the state it was before the resize.
```

Comment:
It's "in case", not "incase". Also, split the second sentence into two:

```markdown
{{brand}} Cloud will now make a restore point in case the resize process fails.
It would then restore your server to the state it was before the resize.
```